apps/web/app/(use-page-wrapper)/settings/(settings-layout)/developer/webhooks/(with-loader)/loading.tsx

```diff
@@ -1,14 +1,5 @@
-import SettingsHeader from "@calcom/features/settings/appDir/SettingsHeader";
-import { SkeletonLoader } from "@calcom/features/webhooks/pages/webhooks-view";
+import { SkeletonLoader } from "@calcom/features/webhooks/pages/webhooks-skeleton";
 
 export default function Loading() {
-  return (
-    <SettingsHeader
-      title="Webhooks"
-      description="Receive meeting data in real-time when something happens in Cal.com"
```

Comment:
Rather than displaying English in Loading component, we can display translated words